### DISCUSSION TITLE
Deploy functions get failed every time in Parallel GPT. Help me in this regard please.

### DISCUSSION THREAD

**Discussion by vicki203** [AntlerVC/firetable#1376] [Support / Q&A] [answered]
Every time I do Deploy Functions in the ParallelGpt template it gets failed. please help me in this regard.

**Comment by hrutik7** [comment, marked as answer]
Hi @vicki203 ,
if your automation deployment is failing via the setup you can simply proceed with the process. Once your table is created, you can manually make a small change in the scripts (adding a comment or a logging statement) and re-deploy your derivatives and extensions.
